plugins/utils/todo-comments.nix

```diff
@@ -226,16 +230,27 @@ in {
             before
             keyword
             after
-            pattern
             exclude
             ;
+          pattern =
+            helpers.ifNonNull' cfg.highlight.pattern
+            (helpers.mkRaw cfg.highlight.pattern);
```

Comment:
I don't really want us to use a mkRaw here, because if people hardcode a pattern as a nix string ex: " TODO:", then it would break because we dumped this as a pure lua code, whereas an user would expect the string to be quoted.

I think we should add `[[` and `]]` to the strings, and mention it in the documentation.

Reply:
Now it should be better.